Crash consistent VM restore points (preview)/README.md

```diff
@@ -2,12 +2,12 @@
 A crash consistent VM restore point stores the VM configuration and point-in-time write-order consistent snapshots for all managed disks attached to a Virtual Machine. This is same as the status of data in the VM after a power outage or a crash.
 
 ## Get started
-You can now create multi-disk crash consistent restore points for your Azure VMs and use these restore points for backup and/or disaster. Crash consistent VM restore points are available in the following regions: EAST US 2 EUAP. If you want know more about VM restore points, please review our [VM restore points public documentation](https://docs.microsoft.com/en-us/azure/virtual-machines/virtual-machines-create-restore-points). Your subscription needs to be whitelisted for using crash consistent restore points so please reach out at VMRestorepoints@microsoft.com to participate in the private preivew.
+You can now create multi-disk crash consistent restore points for your Azure VMs and use these restore points for backup and/or disaster recovery. Crash consistent VM restore points are available in the regions mentioned below. If you want to know more about VM restore points, please review our [VM restore points public documentation](https://docs.microsoft.com/en-us/azure/virtual-machines/virtual-machines-create-restore-points). Your subscription needs to be whitelisted for using crash consistent restore points so please reach out at VMRestorepoints@microsoft.com to participate in the private preivew.
```

Comment:
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/whitelist We can replace whitelisted with allowlisted.